pac4j-core/src/main/java/org/pac4j/core/http/DefaultAjaxRequestResolver.java

```diff
@@ -30,6 +30,7 @@
     /**
```

Comment:
I think the use of `@Override` allows us not to add `{@inheritDoc}`, doesn't it? The Javadoc comment can be safely removed.